src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -0,0 +1,65 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.spark.config.EncoderConfig;
+import com.spikes2212.command.genericsubsystem.smartmotorcontrollersubsystem.SmartMotorControllerGenericSubsystem;
+import com.spikes2212.util.smartmotorcontrollers.SparkWrapper;
+import edu.wpi.first.wpilibj.DigitalInput;
+
+public class Elevator extends SmartMotorControllerGenericSubsystem {
+
+    public enum ElevatorLevels {
+
+        PROCESSOR(-1), FEEDER(-1), L1(-1), L2(-1), L3(-1), L4(-1);
```

Comment:
according to mechanics, processor and feeder require the same heights as certain levels, add a todo to check this